### PR TITLE
Add performance benchmarks for dataframe arithmetic operations 

### LINKED ISSUE #6826: Add performance benchmarks for dataframe arithmetic operations
Add performance benchmarks for dataframe arithmetic operations

### PR DESCRIPTION
**Pull request by asmirnov82**
fixes #6826

### CODE REVIEW

**Comment by asmirnov82** [comment]
@JakeRadMSFT could you please take a look?

**Comment by JakeRadMSFT** [comment]
On the surface these look fine to me - I'm not sure how we should integrate into our builds though. It would be great to have reports show up in PRs or somewhere ... so we can track  how changes are impacting perf.

File: test/Microsoft.Data.Analysis.PerformanceTests/Microsoft.Data.Analysis.PerformanceTests.csproj [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <ImplicitUsings>disable</ImplicitUsings>
+    <TargetFrameworks>net6.0</TargetFrameworks>
+    <GenerateProgramFile>false</GenerateProgramFile>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="BenchmarkDotNet" Version="0.13.5" />
```
Comment (JakeRadMSFT) [inline test/Microsoft.Data.Analysis.PerformanceTests/Microsoft.Data.Analysis.PerformanceTests.csproj:11]: We should use the same version as the other performance projects - $(BenchmarkDotNetVersion)

Comment (JakeRadMSFT) [reply test/Microsoft.Data.Analysis.PerformanceTests/Microsoft.Data.Analysis.PerformanceTests.csproj:11]: See:

https://github.com/dotnet/machinelearning/blob/main/test/Microsoft.ML.PerformanceTests/Microsoft.ML.PerformanceTests.csproj#L14C58-L14C83

Comment (asmirnov82) [reply test/Microsoft.Data.Analysis.PerformanceTests/Microsoft.Data.Analysis.PerformanceTests.csproj]: @JakeRadMSFT there is an issue with the DotNetBenchmark version 0.12.0, that is currently used in ML .Net. It crashes with  NotSupportedException("Unknown .NET Runtime") on .Net 6.0. 
However. upgrading solution to the latest 0.13.5 leads to build error as Obsolete methods from this library are used in Microsoft.ML.CpuMath.PerformanceTests. 

I am not able to build Native part of ML .Net on my laptop ( due to #6823 ), so I can't verify my changes in the CpuMath.PerformanceTests project (or check if it's working correctly on .Net 6.0). That's why I used the latest version only for the DataFrame project.

Could you please help me with building native dlls for the project?

Comment (JakeRadMSFT) [reply test/Microsoft.Data.Analysis.PerformanceTests/Microsoft.Data.Analysis.PerformanceTests.csproj]: Yeah let me take a look!

**Review by JakeRadMSFT — APPROVED** [review]

**Pull request merged**